src/main/java/thrift/commons/util/DateUtils.java

```diff
@@ -0,0 +1,286 @@
+package thrift.commons.util;
+
+import java.util.Calendar;
+import java.util.Date;
+
+/**
+ * A class to ease Date comparisons.
+ * Source: http://www.java2s.com/Code/Java/Data-Type/Checksifacalendardateistoday.htm
+ */
+public class DateUtils {
+
+    /** The maximum date possible. */
+    public static final Date MAX_DATE = new Date(Long.MAX_VALUE);
+
+    /**
+     * <p>Checks if two dates are on the same day ignoring time.</p>
```

Comment:
It should have an empty line between description and parameter section.

Reply:
Resolved in 58f47b5